frontend/src/core/term/components/Term.js

```diff
@@ -52,33 +52,36 @@ export default function Term(props: Props) {
         isReadOnlyEditor || !term.translation ? 'cannot-copy' : '';
 
     return (
-        <li
-            className={`term ${cannotCopy}`}
-            onClick={() => copyTermIntoEditor(term.translation)}
-        >
-            <header>
-                <span className='text'>{term.text}</span>
-                <span className='part-of-speech'>{term.partOfSpeech}</span>
-                <a
-                    href={`/${locale}/terminology/common/?string=${term.entityId}`}
-                    onClick={navigateToPath}
-                    className='translate'
-                >
-                    Translate
-                </a>
-            </header>
-            <p className='translation'>{term.translation}</p>
-            <div className='details'>
-                <p className='definition'>{term.definition}</p>
-                {!term.usage ? null : (
-                    <p className='usage'>
-                        <Localized id='term-Term--for-example'>
-                            <span className='title'>E.G.</span>
-                        </Localized>
-                        <span className='content'>{term.usage}</span>
-                    </p>
-                )}
-            </div>
-        </li>
+        <Localized id='terms-Translation--copy' attrs={{ title: true }}>
```

Comment:
String identifiers [should follow this form](https://github.com/mozilla/pontoon/tree/master/frontend#adding-localized-content): `${module}-${ComponentName}--${string-description}`, so in this case `term-Term--copy`.

Reply:
Thanks for sharing the documentation link. Noted :-)